src/Graph.elm

```diff
@@ -699,10 +666,10 @@ mapNodes f =
         empty
 
 
-{-| Maps over edge labels, possibly chaing their types. Leaves the graph
+{-| Maps over edge labels Leaves the graph
 topology intact.
 -}
-mapEdges : (e1 -> e2) -> Graph n e1 -> Graph n e2
+mapEdges : (e -> e) -> Graph n e -> Graph n e
```

Comment:
Is there any reason to make the type more restrictive? With previous type signature people could change the type of edge labels. With this change it won't be possible any more..

Reply:
I agree that this is an unfortunate loss of flexibility.  The issue is that 0.19 no longer allows you to change the type of records in this way (https://github.com/elm/compiler/blob/master/upgrade-docs/0.19.md#stricter-record-update-syntax).

I figured out how to make it work, so I've restored the original type (for mapNodes too)